napari/_qt/qt_viewer.py

```diff
@@ -439,7 +440,9 @@ def _create_canvas(self) -> None:
         self.canvas.events.mouse_double_click.connect(
             self.on_mouse_double_click
         )
-        self.canvas.events.mouse_move.connect(self.on_mouse_move)
+        self.canvas.events.mouse_move.connect(
+            qthrottled(self.on_mouse_move, timeout=5)
```

Comment:
```suggestion
            qthrottled(self.on_mouse_move, timeout=15)
```

not sure we need ~180 fps, the above is ~60 fps

Reply:
In this calculation, you totally omit time for function processing. 

It Qt we handle events in the main thread. So if some calculation is performed then the start of timer is delayed. 

Assume that event happens in mid of the calculation of the function takes 10ms. Then In the case of @brisvag code the event processing was triggered 10ms after the real event and in your case it is 20ms. 

If using 15 ms timeout I expect 30-40 fps, not 60.

Reply:
Makes sense, thanks @Czaki !